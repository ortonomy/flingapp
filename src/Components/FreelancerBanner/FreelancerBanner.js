import React from 'react';
import banner from './FreelancerBanner.module.css';

function FreelancerBanner (props) {
    return (
      <div className={banner.FreelancerBanner}>
        <div className={banner.text}>
          <h2>{props.name}</h2>
          <div className={banner.active}>{props.status}</div>
        </div>
        <div className={banner.title}>{props.title}</div>
      </div>
    )
}

export default FreelancerBanner;
