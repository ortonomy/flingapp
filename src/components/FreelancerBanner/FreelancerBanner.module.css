.FreelancerBanner {
  background-image: radial-gradient(circle at 5% 15%, #ffdd80, #fbc701);
  height: 10em;
  width: 100%;
  padding-left: 3em;
  display: flex;
  flex-flow: column nowrap;
  justify-content: center;
  align-items: flex-start;
}

.text {
  display: flex;
  flex-flow: row nowrap;
  align-items: center;
  justify-content: flex-start;
}

.FreelancerBanner h2 {
  margin: 0;
  font-weight: bold;
  font-size: 3em;
}

.title {
  font-size: 1.3em;
  margin-top: 0.5em;
  text-transform: uppercase;
}

.status {
  width: 6em;
	height: 2em;
	border-radius: 4px;
  margin-left: 1em;
  color: white;
  text-transform: uppercase;
  display: flex;
  align-items: center;
  justify-content: center;
}

.active {
  composes: status;
  background-color: #81d135;
}
