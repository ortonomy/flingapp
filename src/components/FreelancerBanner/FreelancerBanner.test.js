import React from 'react';
import FreelancerBanner from '../FreelancerBanner';


it('renders without crashing', () => {
  const wrapper = shallow(<FreelancerBanner />);
  expect(wrapper).toMatchSnapshot();
});
