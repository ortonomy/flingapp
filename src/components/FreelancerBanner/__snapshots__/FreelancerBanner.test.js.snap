// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FreelancerBanner />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className={undefined}
>
          <h2 />
          <div
                    className={undefined}
          />
</div>,
        <div
          className={undefined}
/>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2 />,
            <div
              className={undefined}
/>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": undefined,
              "className": undefined,
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className={undefined}
>
            <h2 />
            <div
                        className={undefined}
            />
</div>,
          <div
            className={undefined}
/>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2 />,
              <div
                className={undefined}
/>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": undefined,
                "className": undefined,
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
